SupportClient/SupportClient/Network.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  Network.swift
+//  SupportClient
+//
+//  Created by Omar Rasheed on 5/9/20.
+//  Copyright © 2020 Cornell Appdev. All rights reserved.
+//
+
+import Firebase
+import FirebaseFirestore
+import Foundation
+
+struct Network {
+    
+    private let commonPath = "Patch/data"
+    static private let db = Firestore.firestore()
```

Comment:
do we want this to be a singleton or how is this going to be called?

Reply:
Yea. Updated to be singleton (i thiiink)